leetcode/0003.Longest-Substring-Without-Repeating-Characters/3. Longest Substring Without Repeating Characters.go

```diff
@@ -26,25 +26,35 @@ func lengthOfLongestSubstring(s string) int {
 	return result
 }
 
-// 解法二 滑动窗口
-func lengthOfLongestSubstring_(s string) int {
-	if len(s) == 0 {
-		return 0
+// 解法二 滑动窗口-数组桶
+func lengthOfLongestSubstring1(s string) int {
+	right, left, res := 0, 0, 0
+	var m [256]int
```

Comment:
Is it better to name variable m to freq?

Reply:
array's value is indexs, rename variable `m` to `indexes`